File: nVideo/Models/Catalog_Value.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace nVideo.Models
{
    public class Catalog_Value
    {
        [Key]
        public int Id { get; set; }
        public int? IntegerValue { get; set; }
        public string StringValue { get; set; }

        [ForeignKey("Catalog_Attribute")]
        public Catalog_Attribute Attribute { get; set; }
    }
}

File: nVideo_DAL/Entities/Catalog_Value.fs
module nVideo_DAL.Entities.Catalog_Value
open nVideo_DAL.Entities.Catalog_Attribute
open System.ComponentModel.DataAnnotations;
open System.ComponentModel.DataAnnotations.Schema;

type Catalog_Value = {
    [<Key>]
    Id: string
    
    IntegerValue: int
    StringValue: string
    
    [<ForeignKey("Catalog_Attribute")>]
    Attribute: Catalog_Attribute
}